tests/test_config.json

```diff
@@ -11,7 +11,8 @@
       "os_distribution": "centos",
       "os_product_name": "CentOS Linux release 7.0.1406 (Core)",
       "os_disk_configuration": "Standard",
-      "expected_timeout": "19",
+      "timeout_override": "55",
+      "expected_timeout": "51",
```

Comment:
As evidenced by the change, this starts to get a bit messy when the values change. Perhaps we can use a replacement identifier in the JSON file and do a string replace in the test app with the constants before we call `json.load`?  (Might be a follow-up item to track)

Reply:
Makes sense. I create a follow up item for this.